Formula/kamel.rb

```diff
@@ -0,0 +1,38 @@
+class Kamel < Formula
+  desc "Apache Camel K CLI"
+  homepage "https://camel.apache.org/"
+
+  url "https://github.com/apache/camel-k.git",
+    :tag      => "1.0.0-RC2",
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
I have changed this to version 0.3.4 the latest stable one.